docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -265,7 +265,11 @@ proposal because most of the changes described below are a consequence of the wo
     however, using different timeouts for init and join will be enabled only when v1beta1 is removed.
   - `spec.joinConfiguration.discovery.timeout` field has been removed. Use `spec.joinConfiguration.timeouts.tlsBootstrapSeconds` instead.
 - The `spec.useExperimentalRetryJoin` field (deprecated in CAPI v1.2!) has been removed.
-- The following `spec.kubeadmConfigSpec.clusterConfiguration` fields have been removed because they are duplicates
+- The following `spec` fields have been removed because not necessary (Cluster API automatically applies the right gvk when generating kubeadm config files):
```

Comment:
```suggestion
- The following `spec` fields have been removed because they are not necessary (Cluster API automatically applies the right gvk when generating kubeadm config files):
```